Dockerfile-dockerhub

```diff
@@ -13,10 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM confluentinc/cp-kafka-connect-base:5.3.0
+FROM confluentinc/cp-kafka-connect:5.3.1
 
 ENV CONNECT_PLUGIN_PATH="/usr/share/java,/usr/share/confluent-hub-components"
 
-COPY target/components/packages/confluentinc-kafka-connect-datagen-0.1.3.zip /tmp/confluentinc-kafka-connect-datagen-0.1.3.zip
+COPY target/components/packages/confluentinc-kafka-connect-datagen-0.1.4-SNAPSHOT.zip /tmp/confluentinc-kafka-connect-datagen-0.1.4-SNAPSHOT.zip
```

Comment:
It's not ideal to reference a SNAPSHOT release. Should we just release kafka-connect-datagen 0.1.4 and then update this to point to the new released version?

Reply:
@rhauch other confluentinc `.x` branches work the same way with `SNAPSHOT`, e.g. https://github.com/confluentinc/kafka-streams-examples/blob/5.3.x/pom.xml, https://github.com/confluentinc/schema-registry/blob/5.3.x/pom.xml, etc.  That's how our tooling automation works so this repo should be consistent.